appinfo/info.xml

```diff
@@ -2,7 +2,7 @@
 <info>
     <id>security</id>
     <name>Security</name>
-    <description>OwnCloud security app. It harden ownCloud security with several new features like brute force protection, 404 detection, strong password enforcement.</description>
+    <description>ownCloud security app. It harden ownCloud security with several new features like brute force protection, 404 detection, strong password enforcement.</description>

```

Comment:
ownCloud Security app. It hardens ownCloud security with several new features including brute force protection, 404 detection and strong password enforcement.